main/glossary/README.md

```diff
@@ -413,6 +413,10 @@ episode [The Gamesters of Triskelion](https://en.wikipedia.org/wiki/The_Gamester
 A transfer of [amounts](#amounts) between [seats](#seat) within Zoe; i.e. a change in their [allocations](#allocation). When a seat exits, it gets its
 current allocation as a [payout](#payout). 
 
+## Record and Tuple
+
+Records and tuples are immutable objects which can be passed by copy. They only contain values which can be passed by copy such as numbers, strings, or other records and tuples. 
```

Comment:
Hmm, they can contain presences/remotables (what's the right term again?) as values, so we should probably mention that. Otherwise I think this description is misleading.

Reply:
Oh I didn't realize that remotables could be nested inside pass by copy structures! And it is definitely not obvious from the documentation.
https://agoric.com/documentation/guides/js-programming/far.html